### PR TITLE
Basic enemy AI implementation

### PR DESCRIPTION
**Pull request by microfolk**

### CODE REVIEW

File: objects/aiable.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from random import choice, randint
+
+class AIObject:
+
+    # the idea would be to pass the algorithm to the __init__ but I don't know how to do that!
```
Comment (afiuorio) [inline objects/aiable.py]: You could pass functions to functions in python.

File: objects/aiable.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    def act(self):
+        return self.random()
+
+    @staticmethod
+    def random():
```
Comment (afiuorio) [inline objects/aiable.py]: I don't really like this approach. Why we don't have classes that represents a specific Entity and put the IA logic in them?

File: objects/descriptionobject.py [hunk 3]
```diff
@@ -1,5 +1,9 @@
 class DescriptionObject:
 
     def __init__(self, name, description):
+        self.id = id
```
Comment (afiuorio) [inline objects/descriptionobject.py]: id is a standard function of Python, so you are assigning to the id field of the object the reference to the id function (which I doubt is something you want). Also, this id field isn't currently used anywhere, so it should be removed.

Comment (microfolk) [reply objects/descriptionobject.py]: Oh wow that's something that shouldn't be there I completely missed it

**Pull request merged**